helm/alfresco-content-services/templates/config-repository.yaml

```diff
@@ -61,6 +61,45 @@ data:
       -Ds3.{{ $key }}={{ $val }}
       {{- end }}
       {{- end }}
+      {{- if .Values.email.server.enabled }}
+      -Demail.server.enabled={{ .Values.email.server.enabled }}
+      -Demail.server.port={{ .Values.email.server.port }}
+      -Demail.server.domain={{ .Values.email.server.domain }}
+      -Demail.server.enableTLS={{ .Values.email.server.enableTLS }}
+      -Demail.server.hideTLS={{ .Values.email.server.hideTLS }}
+      -Demail.server.requireTLS={{ .Values.email.server.requireTLS }}
+      -Demail.server.auth.enabled={{ .Values.email.server.auth.enabled }}
+      -Demail.server.connections.max={{ .Values.email.server.connections.max }}
+      -Demail.server.allowed.senders={{ .Values.email.server.allowed.senders }}
+      -Demail.server.blocked.senders={{ .Values.email.server.blocked.senders }}
+      -Demail.inbound.enabled={{ .Values.email.inbound.enabled }}
+      -Demail.inbound.unknownUser={{ .Values.email.inbound.unknownUser }}
+      -Demail.inbound.emailContributorsAuthority={{ .Values.email.inbound.emailContributorsAuthority }}
+      -Demail.handler.folder.overwriteDuplicates={{ .Values.email.handler.folder.overwriteDuplicates }}
+      -Dmail.encoding={{ .Values.mail.encoding }}
+      -Dmail.host={{ .Values.mail.host }}
+      -Dmail.port={{ .Values.mail.port }}
+      -Dmail.protocol={{ .Values.mail.protocol }}
+      -Dmail.username={{ .Values.mail.username }}
+      -Dmail.from.default={{ .Values.mail.from.default }}
+      -Dmail.from.enabled={{ .Values.mail.from.enabled }}
+      -Dmail.smtp.auth={{ .Values.mail.smtp.auth }}
+      -Dmail.smtp.debug={{ .Values.mail.smtp.debug }}
+      -Dmail.smtp.starttls.enable={{ .Values.mail.smtp.starttls.enable }}
+      -Dmail.smtp.timeout={{ .Values.mail.smtp.timeout }}
+      -Dmail.smtps.auth={{ .Values.mail.smtps.auth }}
+      -Dmail.smtps.starttls.enable={{ .Values.mail.smtps.starttls.enable }}
+      -Dimap.server.enabled={{ .Values.imap.server.enabled }}
+      -Dimap.server.port={{ .Values.imap.server.port }}
+      -Dimap.server.host={{ .Values.imap.server.host }}
+      -Dimap.server.imap.enabled={{ .Values.imap.server.imap.enabled }}
+      -Dimap.server.imaps.enabled={{ .Values.imap.server.imaps.enabled }}
+      -Dimap.server.imaps.port={{ .Values.imap.server.imaps.port }}
+      -Dimap.mail.from.default={{ .Values.imap.mail.from.default }}
+      -Dimap.mail.to.default={{ .Values.imap.mail.to.default }}
+      -Dcsrf.filter.enabled=false
```

Comment:
Do you intend to put the "csrf.filter.enabled" and "system.usages.enabled" flags inside the  "{{- if" statement?

Reply:
I've seen these 2 params in [prerequisite-for-acs-60-using-docker-images](https://github.com/Alfresco/alfresco-tas-email#prerequisite-for-acs-60-using-docker-images) and rendering it inside the if/end block [here](https://github.com/Alfresco/acs-deployment/blob/8fa2f04fdf440c3efdee069e1458fff50fcc344b/helm/alfresco-content-services/templates/config-repository.yaml#L64-L102)

Reply:
Hi Abdul, I reached out to Alex to double check on this, his response was: 

> Hi, disabling the csrf filter is a security problem.
> it should be configured with valid origin and referer instead

So as I suspected we're going to need to address this as we can't introduce a security hole when someone wants to use email.